### PR TITLE
Fix Part of #2833 : Add missing named argument in RecentlyPlayedFragmentTest

### PR DESCRIPTION
**Pull request by techjd**
## Explanation
Fixes Part of #2833 Added Missing Named Argument in RecentlyPlayedFragmentTest

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @techjd 

@rt4914 Can we have a difference between 

```kotlin
  private val internalProfileId = 0
  private lateinit var profileId: ProfileId
```

as we know, `ProfileId` is a model class and `internalProfileId` is the index of profile in the list. So, could you suggest a better naming here?

**Comment by rt4914** [comment]
> LGTM, Thanks @techjd
> 
> @rt4914 Can we have a difference between
> 
> ```kotlin
>   private val internalProfileId = 0
>   private lateinit var profileId: ProfileId
> ```
> 
> as we know, `ProfileId` is a model class and `internalProfileId` is the index of profile in the list. So, could you suggest a better naming here?

@anandwana001 Actually this was exactly the naming convention we decided way back.

If `int` type we use `internalProfileId` considering that ProfileId model does define this attribute too.
If `ProfileId` type simply use `profileId`.

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 1]
```diff
@@ -148,16 +148,16 @@ class RecentlyPlayedFragmentTest {
 
   private fun createRecentlyPlayedActivityIntent(profileId: Int): Intent {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Suggest renaming `profileId` to `internalProfileId`

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 2]
```diff
@@ -148,16 +148,16 @@ class RecentlyPlayedFragmentTest {
...
+      internalProfileId = profileId
     )
   }
 
   @Test
   fun testRecentlyPlayedTestActivity_clickOnToolbarNavigationButton_closeActivity() {
     activityTestRule.launchActivity(
       createRecentlyPlayedActivityIntent(
-        internalProfileId
+        profileId = internalProfileId
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Write it as `internalProfileId = internalProfileId` everywhere.

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 3]
```diff
@@ -186,16 +186,16 @@ class RecentlyPlayedFragmentTest {
   fun testRecentlyPlayedTestActivity_sectionDividerIsNotDisplayed() {
     fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_UPTIME_MILLIS)
     storyProgressTestHelper.markRecentlyPlayedFractionsStory0Exp0(
-      profileId,
+      profileId = profileId,
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt:189]: This one is correct. Similar all functions which are part of `storyProgressTestHelper ` are correct.

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 4]
```diff
@@ -789,7 +854,9 @@ class RecentlyPlayedFragmentTest {
       )
       onView(withId(R.id.ongoing_story_recycler_view)).check(
         hasGridItemCount(
-          1, 3
+
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Remove this blank line

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL, suggested changes.

**Comment by anandwana001** [comment]
@techjd any update on this?

**Comment by techjd** [comment]
> @techjd any update on this?

@anandwana001 You can take a look at the last 2 commits . It's done .

**Comment by anandwana001** [comment]
@techjd please reply to all the conversation started by Rajat, with something like done, or if anything left to do please mention it.

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 1 above)
Comment (techjd) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Done!

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 2 above)
Comment (techjd) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Done!

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 4 above)
Comment (techjd) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Done!

**Comment by techjd** [comment]
@anandwana001 Done!

**Comment by anandwana001** [comment]
This one too - https://github.com/oppia/oppia-android/pull/2897#discussion_r594485155

**Comment by techjd** [comment]
@anandwana001 That was a reference shown for an explanation, that's why I didn't replied to that conversation.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**